sdk/eventgrid/azure-eventgrid/samples/champion_scenarios/cs3_consume_system_events.py

```diff
@@ -11,23 +11,18 @@
     python cs3_consume_system_events.py
 """
 import os
+import json
 from azure.eventgrid import EventGridConsumer
 
 consumer = EventGridConsumer()
 
+with open('./cs3_event_grid_event_system_event.json', 'r') as f:
+    eg_event_received_message = json.loads(f.read())
```

Comment:
or `json.load(f)` 

I think here my comment is just FYI, since it's mostly the same thing